src/components/action-popover/action-popover.spec.tsx

```diff
@@ -180,6 +181,249 @@ describe("ActionPopover", () => {
     );
   }
 
+  function renderActionPopoverWithNoIconsOrSubmenus(
+    props = {},
+    renderer = mount
+  ) {
+    const defaultProps = {
+      children: [
+        <ActionPopoverItem key="item-1" onClick={() => {}}>
+          Business
+        </ActionPopoverItem>,
+        <ActionPopoverItem key="item-2" onClick={() => {}}>
+          Email Invoice
+        </ActionPopoverItem>,
+        <ActionPopoverItem key="item-3" onClick={() => {}}>
+          Print Invoice
+        </ActionPopoverItem>,
+        <ActionPopoverItem key="item-4" onClick={() => {}}>
+          Download PDF
+        </ActionPopoverItem>,
+        <ActionPopoverItem key="item-5" onClick={() => {}}>
+          Download CSV
+        </ActionPopoverItem>,
+        <ActionPopoverDivider key="item-6" />,
+        <ActionPopoverItem key="item-7" onClick={() => {}}>
+          Delete
+        </ActionPopoverItem>,
+        <ActionPopoverItem key="item-8" onClick={() => {}}>
+          Return Home
+        </ActionPopoverItem>,
+        null,
+        undefined,
+      ],
+      ...props,
+    };
+
+    renderer(
+      <ThemeProvider theme={mintTheme}>
```

Comment:
**question** - do we need the `mintTheme` (or a `ThemeProvider` at all) for any of these tests? Pretty sure that's a legacy thing. I do see it was in use before for some tests in this file but it looks like you're using it in a lot more now. Now would be a good time to remove all mentions of it if it's not actually needed.

I'm only making this comment once but of course it's used in several of these functions, the same comment applies to each.